packages/flutter/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove this?

Reply:
Sorry 🤦‍♂️, fixed.

Reply:
I think you wanted to delete your edit, but not the entire file  :)

Reply:
It seems like now all is fine :)